fem/fespace.hpp

```diff
@@ -975,6 +977,25 @@ class QuadratureSpace
 
    /// Write the QuadratureSpace to the stream @a out.
    void Save(std::ostream &out) const;
+
+   /// Returns a name that can be used to reconstruct the QuadratureSpace
+   const char *Name() const { return qs_name; }
+
+   /** @brief Factory method: return a newly allocated QuadratureSpace
+       according to the given basis name.
+       @param[in] name The basis name to generate a QuadratureSpace for
+       @param[in] mesh The mesh to use when constructing the QuadratureSpace */
+   /**
+
+   | QSpace Basis Name | Notes |
+   | :------: | :---: |
+   | QF_Default_[ORDER] | Default quadrature type |
+
+   | Tag | Description |
+   | :------: | :--------: |
+   | [ORDER]  | Approximation order of the elements (P0, P1, P2, ...) |
```

Comment:
Order here isn't the approximate order of the element. Rather, it is the order of the integration rule being used for the `QuadratureSpace`.